services/gcp/cloudfunctions/functions.yml

```diff
@@ -52,19 +52,11 @@ privileges:
       Also allows for DOS via spamming executions and data injection via execution with fake parameters.
   create:
     risks:
-      - discovery:account
-      - discovery:infra
-      - discovery:data
-      - exfiltration:data
-      - takeover:account
       - impact:spend
       - impact:hijack
     notes: >-
       Creating a cloud function requires permissions on the cloud functions runtime service account.
-      It then allows you to export credentials to the service account.
```

Comment:
Similar to what we do for compute.instances.create, can we indicate how credentials can be read, and why that isn't a risk?